locator/token_metadata.cc

```diff
@@ -480,6 +487,7 @@ future<> token_metadata_impl::update_normal_tokens(const std::unordered_map<inet
     if (should_sort_tokens) {
         sort_tokens();
     }
+    update_normal_token_owners();
```

Comment:
nit: better be done before sorting the tokens, right after we finish messing with _token_to_endpoint_map.

Reply:
OK.